From the GitHub Actions checks:

```
src/imcflibs/imagej/misc.py:19: in <module>
    from org.scijava.widget import WidgetStyle
E   ModuleNotFoundError: No module named 'org'
```

Looks like you need to update `imcf-fiji-mocks` now (and bump its version here) to provide the necessary `org.scijava` classes.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -20,7 +20,7 @@ version = "0.0.0"
 # - or: python = ">=3.10"
 
 [tool.poetry.dependencies]
-imcf-fiji-mocks = ">=0.10.0"
+imcf-fiji-mocks = ">=0.12.0"
 python = ">=2.7"
 python-micrometa = "^15.2.2"
 sjlogging = ">=0.5.2"

--- a/src/imcflibs/imagej/misc.py
+++ b/src/imcflibs/imagej/misc.py
@@ -728,7 +728,7 @@ def save_script_parameters(destination, save_file_name="script_parameters.txt"):
     # Get the ScriptModule object from globals made by Fiji
     module = globals().get("org.scijava.script.ScriptModule")
     if module is None:
-        print("No ScriptModule found — skipping saving script parameters.")
+        print("No ScriptModule found - skipping saving script parameters.")
         return
 
     destination = str(destination)

